src/switchroot/ostree-remount.c

```diff
@@ -112,6 +112,11 @@ main(int argc, char *argv[])
   bool sysroot_configured_readonly = unlink (_OSTREE_SYSROOT_READONLY_STAMP) == 0;
   do_remount ("/sysroot", !sysroot_configured_readonly);
 
+  /* And also make sure to make /etc rw again. We make this conditional on
```

Comment:
Hmm but...this case can only happen if the rootfs was mounted read-only to start, which isn't the case in FCOS right?  We mount writable to start because Ignition needs to write to it.

Reply:
When we remount `/sysroot` read-only just above this line, it also makes `/etc` and `/var` read-only.